src/reactor/reactor.spec.ts

```diff
@@ -541,6 +541,175 @@ describe('reactor/reactor with cycles', () => {
     });
 });
 
+describe('reactor/reactor error handling', () => {
+    let a$: Atom<string>;
+    let d$: Derivable<string>;
+
+    beforeEach('setup atom and derivable', () => {
+        a$ = atom('no error');
+        d$ = a$.derive(v => {
+            if (v === 'error in derivation') {
+                throw new Error(v);
+            }
+            return v;
+        });
```

Comment:
May want to put an extra `derive` here, to explicitly support errors thrown in any `derive` step.